cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go

```diff
@@ -970,10 +977,178 @@ var InstanceTypes = map[string]*InstanceType{
 		GPU:          0,
 		Architecture: "arm64",
 	},
-	"cc2.8xlarge": {
```

Comment:
Surprised that this is disappearing 5 years after the instances were shut down, though weirdly can only seem to find this: https://developer.basespace.illumina.com/docs/content/documentation/native-apps/cc2-deprecation